staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/validator.go

```diff
@@ -74,7 +74,7 @@ func (c *CELValidatorCompiler) BindingMatches(a admission.Attributes, o admissio
 		return true, nil
 	}
 	criteria := matchCriteria{constraints: binding.Spec.MatchResources}
-	return c.Matcher.Matches(a, o, &criteria)
+	return c.Matcher.Matches(a, o, &criteria, true)
```

Comment:
It surprised me that we have these `{Binding,Definition}Matches` functions and that we now also have a boolean being passed into Matches. It looks like we're trying to distinguish between definitions and bindings using two mechanisms when one would be sufficient.

Reply:
Right, I understand _how_ the fix works, and it does fix the problem.  My question is _why_ are we structuring the code this way?

Reply:
I believe this is resolved now that absent resourceRules are considered a wildcard match.